src/aiida/orm/nodes/data/array/trajectory.py

```diff
@@ -131,14 +137,20 @@ def set_trajectory(self, symbols, positions, stepids=None, cells=None, times=Non
         :param velocities: if specified, must be a float array with the same
                       dimensions of the ``positions`` array.
                       The array contains the velocities in the atoms.
+        :param pbc: periodic boundary conditions of the structure. Should be a list of
+            length three with booleans indicating if the structure is periodic in that
+            direction. The same periodic boundary conditions are set for each step.
 
         .. todo :: Choose suitable units for velocities
         """
         import numpy
 
-        self._internal_validate(stepids, cells, symbols, positions, times, velocities)
-        # set symbols as attribute for easier querying
+        pbc = pbc or [True, True, True]
```

Comment:
```suggestion
        pbc = pbc or [True, True, True]
```
This default makes me uneasy. Should we instead look at the `cells` and see if it is defined and non-zero?

Reply:
Sensible check! My MC3D bias is showing again ^^

Reply:
> if it is defined and non-zero?

Can the `cells` be defined as zero? 🤔

Reply:
Do not know when I would use a structure without specifying cells, but agree to consider them `False` if not specified

Reply:
Apparently you can set e.g. `cell = [[0, 0, 0], [0, 1, 0], [0, 0, 1]]`, but I'm not sure that's sensible? @danielhollas right now I've adapted the default to be `[False, False, False]` in case `cells=None`. Is that sufficient? Or what did you have in mind?